api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1626,7 +1626,7 @@ public ObjectStat statObject(String bucketName, String objectName)
    *
    * <pre>Example:{@code
    * ObjectStat objectStat =
-   *     minioClient.statObject("my-bucketname", "my-objectname", ssec);
+   *     minioClient.statObject("my-bucketname", "my-objectname", sse);
```

Comment:
This is still `ssec`. You could revert this change.

Reply:
Will revert